### PR TITLE
Add team D information - 이혜연

### PR DESCRIPTION
**Pull request by hyeyeon-sun**
## Description
team-D-description.md 파일에 팀원 이혜연의 정보를 업데이트 합니다.

## Context
Challengers:1주차
 - 환경설정 및 Git/GitHub 을 통한 피드백 및 PR 규칙 익히기

## Checklist:

[x] Updated tests for this change.
[x] Tested the changes locally.
[x] Updated documentation.

### CODE REVIEW

**Comment by lizable** [comment]
@hyeyeon-sun Did you have any specific reasons to close this PR?

**Comment by hyeyeon-sun** [comment]
> @hyeyeon-sun Do you have any specific reasons to close this PR?

After the fetch, there was a change history in the main branch, resulting in conflic. So, I'll try to fix it and create a PR again.

**Comment by hyeyeon-sun** [comment]
> @hyeyeon-sun Do you have any specific reasons to close this PR?
 
@lizable  I found a way to resolve the conflict on github, so I reopened the PR and resolved the conflict!

File: team-D-description.md [hunk 1]
```diff
@@ -9,8 +9,11 @@
...
 간단 자기소개 : 안녕하세요. 컴퓨터공학부 4학년에 재학중인 24살 이희광입니다. 저는 벡엔드와 빅데이터와 같은 인공지능에 관심을 가지고 있습니다. 그래서 다음학기부터는 자대 석사 예약입학이 진행되어 대학원 공부도 시작될 예정이며 스타트업에서 안드로이드 개발자 업무도 병행하고 있습니다.  개발을 하면서 깃과 깃허브는 곧 잘 사용하긴했지만 제가 사용하는 기능이 매우 제한적이였기에 제대로 사용하는것같다는 느낌을 받지 못했는데 그래서 이번에 이 오픈소스 프로젝트에 참여하여 제대로 공부해보고자 하였습니다. 다행히 합격하게 되어 멘티분들과 함께 공부할수있게되고 멘토분들의 지도를 받을 수 있는것에 매우 감사하게 생각합니다.   
 
-
-
 이름 : 백종현   
 관심 컴포넌트 : backend.ai-manager, backend.ai-webserver
 간단 자기소개 : 안녕하세요. 올해 2월에 졸업하고 취업을 하게되어 개발자로 일하고 있는 백종현입니다. 학생때는 간단한 개발만 해왔고 일한지 6개월 밖에 되지 않기도 했기 때문에, 스스로 "개발자"라고 부르는 것을 부끄러워 하는 사람입니다. 스스로 개발자라고 당당하게 말할 수 있는 사람이 되는 것이 목표입니다. 현재 개발한지 오래된 프로젝트에서 유지보수성 개발을 하고 있기 때문에, 주로 레거시 기반의 자바 코드를 작성하고 있습니다. 이번 기회를 통해 회사일에서 벗어나 파이썬 개발 등 다양한 경험을 해보고 싶습니다. 인상이 좋은 편이 아니라 다가가기 어려워 보겠지만, 많이 열려있는 사람이니 편하게 대해주셨으면 좋겠습니다:)
+
+이름 : 이혜연   
+관심 컴포넌트 : backend.ai-manager, backend.ai-webui, backend.ai.web-server
```
Comment (lizable) [inline team-D-description.md:17]: @hyeyeon-sun Please update the name of some components, for example `backend.ai.web-server`.

**Review by lizable — CHANGES_REQUESTED** [review]

File: team-D-description.md (hunk 1 above)
Comment (hyeyeon-sun) [reply team-D-description.md]: I've fixed it :)

**Comment by lizable** [comment]
@hyeyeon-sun You need to fix the conflicts.

**Review by lizable — APPROVED** [review]
LGTM!

**Pull request merged**